lms/djangoapps/instructor/enrollment.py

```diff
@@ -259,14 +264,19 @@ def reset_student_attempts(course_id, student, module_state_key, requesting_user
             module_state_key.to_deprecated_string(),
         )
 
-    module_to_reset = StudentModule.objects.get(
-        student_id=student.id,
-        course_id=course_id,
-        module_state_key=module_state_key
-    )
+    try:
+        module_to_reset = StudentModule.objects.get(
+            student_id=student.id,
+            course_id=course_id,
+            module_state_key=module_state_key
+        )
+    except StudentModule.DoesNotExist, ex:
```

Comment:
@efischer19 and I were discussing the fact that this error currently occurs in production if a staff member attempts to delete student state for a problem with no student state (for example a text block). There is error logging earlier in the function, so it seems appropriate to log it here as well.